packages/web-twig/composer.json

```diff
@@ -1,69 +1,78 @@
 {
   "name": "lmc/spirit-web-twig-bundle",
-  "version": "3.1.0",
-  "type": "symfony-bundle",
   "description": "This is Symfony bundle with twig implementation of the Spirit Design System components and extends the twig implementation with an HTML-like syntax.",
   "license": "MIT",
+  "type": "symfony-bundle",
+  "version": "3.1.0",
   "authors": [
     {
       "name": "LMC, s.r.o."
     }
   ],
-  "autoload": {
-    "psr-4": {
-      "Lmc\\SpiritWebTwigBundle\\": "src/"
-    }
-  },
-  "autoload-dev": {
-    "psr-4": {
-      "Lmc\\SpiritWebTwigBundle\\": "tests/"
-    }
-  },
   "require": {
     "php": "^8.1",
+    "ext-simplexml": "*",
     "lmc/twigx-bundle": "^3.0.0 !=3.2.0",
     "symfony/config": "^4.4 || ^5.4 || ^6.1",
     "symfony/dependency-injection": "^4.4 || ^5.4 || ^6.1",
     "symfony/http-foundation": "^4.4 || ^5.4 || ^6.1",
     "symfony/http-kernel": "^4.4 || ^5.4 || ^6.1",
     "symfony/polyfill-php80": "^1.23",
-    "twig/twig": "^2.12.5 || ^3.0.0",
-    "ext-simplexml": "*"
+    "twig/twig": "^2.12.5 || ^3.0.0"
   },
   "require-dev": {
-    "phpunit/phpunit": "^9",
-    "mockery/mockery": "^1.5",
+    "ergebnis/composer-normalize": "^2.42",
     "lmc/coding-standard": "^4.1",
-    "symfony/yaml": "^4.4 || ^5.4 || ^6.1",
+    "mockery/mockery": "^1.5",
+    "phpstan/extension-installer": "^1.1",
     "phpstan/phpstan": "^1.2",
     "phpstan/phpstan-mockery": "^1.0",
-    "phpstan/extension-installer": "^1.1",
     "phpstan/phpstan-symfony": "^1.0",
-    "spatie/phpunit-snapshot-assertions": "^4.2.12"
+    "phpunit/phpunit": "^9",
+    "spatie/phpunit-snapshot-assertions": "^4.2.12",
+    "symfony/yaml": "^4.4 || ^5.4 || ^6.1"
+  },
+  "autoload": {
+    "psr-4": {
+      "Lmc\\SpiritWebTwigBundle\\": "src/"
+    }
+  },
+  "autoload-dev": {
+    "psr-4": {
+      "Lmc\\SpiritWebTwigBundle\\": "tests/"
+    }
+  },
+  "config": {
+    "allow-plugins": {
+      "dealerdirect/phpcodesniffer-composer-installer": true,
+      "ergebnis/composer-normalize": true,
+      "phpstan/extension-installer": true,
+      "symfony/flex": true,
+      "symfony/runtime": true
+    }
   },
   "scripts": {
+    "analyze": [
+      "@ecs",
+      "@phpstan"
+    ],
     "ecs": "vendor/bin/ecs check --no-progress-bar --ansi ",
     "ecs:fix": "vendor/bin/ecs check --no-progress-bar --ansi --fix",
+    "lint": [
```

Comment:
I am not quite sure about the `lint` name. This script only validates and normalizes the `composer.json` file, on the other hand I think the linting is a more general term for the code checks. And, in this case also the `ecs` is considered as lint. I am not sure what is now a commonly used practice in PHP projects. I consider everything that checks the code as a linting tool.